nupic/regions/SPRegion.py

```diff
@@ -640,24 +588,17 @@ def _doBottomUpCompute(self, rfInput, resetSignal):
 
     # Run SFDR bottom-up compute and cache output in self._spatialPoolerOutput
```

Comment:
> Is this comment obsolete?

Not, is not. Because the activity that it is refering (`compute and cache output`) continue being performed (https://github.com/david-ragazzi/nupic/blob/remove_old_SPs/nupic/regions/SPRegion.py#L596).